src/coreclr/jit/compiler.hpp

```diff
@@ -4188,6 +4188,15 @@ unsigned Compiler::GetSsaNumForLocalVarDef(GenTree* lcl)
     }
 }
 
+inline bool Compiler::PreciseRefCountsRequired()
```

Comment:
Would it make sense to introduce `PreciseRefCountsRequired` upstream? It looks like general goodness.

Reply:
I'm happy to create the PR to see what others think.  Just leave
https://github.com/dotnet/runtimelab/blob/9ff990c48ccae699b4ef6faeda70672d0e9b22c5/src/coreclr/jit/liveness.cpp#L2185-L2187
out?

Reply:
Yep